packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/About.1.doc.mdx

```diff
@@ -30,17 +98,40 @@ This component requires the following peer dependencies be installed in your app
 | react-dom | ^16.8.5 |
 | react-intl | ^2.8.0 |
 
-<!-- AUTO-GENERATED-CONTENT:END -->
+## Properties
+
+### FolderTree Props Table
+<FolderTreePropsTable />
```

Comment:
I know this table is auto generated but is there way to update the description for the children prop to not include FolderTree.Items?  So it looks like something like "List of items to be displayed as content within the Folder Tree component."

Reply:
Yeah, we can update that from the component. My thought with calling out "FolderTree.Items" specifically is so that consumers know that ONLY FolderTreeItems should be used and not some other React component - do we think that's clear enough if we reword it to just "items"?

Reply:
Updated: [18cf213](https://github.com/cerner/terra-framework/pull/1973/commits/18cf213eb5eabfc53af77417b4a42ba34b3520f7)

Reply:
@sycombs  Thanks for making the update. For where it was being used I think it is clear just stating items as I usually look at field in the table to have a non-technical explanation. Plus you call out FolderTreeItem in the Basic Folder Tree example so I think that is sufficient.